.github/workflows/ci.yml

```diff
@@ -30,8 +30,11 @@ jobs:
 
```

Comment:
you need fast tests, use `FAST_LINT_TEST_CASES=1`. with that, is there still value behind using ParaTest that justify effort to make them work properly?

Reply:
Yeah, I saw #5206 (it actually made me to check ParaTest), but I believe `FAST_LINT_TEST_CASES=1` and ParaTest are not interchangeable, rather they can stack together? ParaTest takes advantage of multiple cores available and runs tests in parallel so it will be faster that regular PHPUnit, regardless of which linter is used.

Reply:
11k of tests is [an error](https://github.com/paratestphp/paratest/issues/749). It's hard to compare `FAST_LINT` with ParaTest when we don't know what the execution time is for full suite, but I believe it's worth anyway, I'm pretty sure parallel run will be faster.

Reply:
it will. if you solve that error in 5 minutes, it's likely great to have paratests integrated. but if you would need to invest 4 weeks to solve it, likely not the best value for that effort ;)

Reply:
After talk with @Jean85 I've switched to [Paraunit](https://github.com/facile-it/paraunit) and locally I was able to execute whole suite and it was much faster:

> Execution time -- 00:03:57
> 
> Executed: 464 test classes, 32519 tests

But with some failures, which you can see in [this run](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/4619270855). It probably depends on the amount of available cores.

Anyway, total duration of current pipeline is [3h 21m 37s](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/4600881262/usage) (longest job: [25m 32s](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/4600881262/jobs/8128045584)) while with Paraunit it's [2h 6m 12s](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/4619270855/usage) (longest job: [14m 15s](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/4619270855/jobs/8167798381)).

Reply:
FYI:

> uh the fatal il CI is easy to fix: you need to rename the abstract test, if they match `*Test.php` it will mess up PHPUnit loader. that's even deprecated in PHPUnit 10.

Reply:
FYI: failure related to `AbstractSetTest` was fixed by renaming this class to `AbstractSetTestCase`. Failure related to `AbstractSmokeTest` was probably fixed at some point somewhere else, because it just disappeared after rebase.